src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -18,16 +18,39 @@ public static class AzureHelper
     {
         private static string _storageApiVersion = "2018-02-01";
 
-        public static async Task<Site> GetFunctionApp(string name, string accessToken, string managementURL)
+        public static async Task<Site> GetFunctionApp(string name, string accessToken, string managementURL, string defaultSubscription = null)
         {
+            if (!string.IsNullOrEmpty(defaultSubscription))
+            {
```

Comment:
Is the idea now that we will only look at one subscription (default or specified) for all Azure actions?

Reply:
No, it's just to check that subscription first. I have about 50 subs in my account :). Everytime I do a publish it searches all 50 subs for the function app (in order) and my function app happens to be in sub number 40something. This will just put the default sub on the top of the list to check. If there is no default sub set, it'll just ignore it

Reply:
Oh right! I misread and didn't realize that the search actually goes through the others too. This is a great enhancement actually. It takes forever for me to find my function app as well. Thanks!